test/quantization/test_quantize.py

```diff
@@ -680,59 +678,15 @@ def checkQuantized(model):
         checkQuantized(model)
 
     def test_quantized_rnn(self):
-        r"""Test execution and serialization for dynamic quantized lstm modules on int8 and fp16
+        r"""Test dynamic quantization, scriptability and serialization for dynamic quantized lstm modules on int8 and fp16
```

Comment:
There is a test below `test_per_channel_lstm_quantize` for per-channel quantization. Should we combine the two here and go over both qconfigs?

Reply:
Yes, I will combine the tests